c_emulator/riscv_callbacks_rvfi.cpp

```diff
@@ -3,41 +3,41 @@
 #include <stdlib.h>
 #include <vector>
 #include <inttypes.h>
+#include <optional>
+#include "rvfi_dii.h"
+
+extern std::optional<rvfi_handler> rvfi;
 
 // Implementations of default callbacks for RVFI.
 // The model assumes that these functions do not change the state of the model.
 void rvfi_callbacks::mem_write_callback(const char *, sbits paddr,
                                         uint64_t width, lbits value)
 {
   if (config_enable_rvfi) {
```

Comment:
Isn't this always true by definition when the callbacks are registered now? I think we can drop the if.